lib/commons/text/is-human-interpretable.js

```diff
@@ -13,42 +13,61 @@ import sanitize from './sanitize';
 function isHumanInterpretable(str) {
   /**
    * Steps:
-   * 1) Check for single character edge cases
+   * 1) Early escape if string is empty
+   *
+   * 2) Check for single alpha character edge cases
+   *
+   * 3) Check for symbolic text character edge cases
    * 		a) handle if character is alphanumeric & within the given icon mapping
-   * 					eg: x (close), i (info)
+   * 					eg: 'aA' for toggling capitalization
    *
-   * 3) handle unicode from astral  (non bilingual multi plane) unicode, emoji & punctuations
+   * 4) handle unicode from astral (non bilingual multi plane) unicode, emoji & punctuations
    * 					eg: Windings font
    * 					eg: '💪'
    * 					eg: I saw a shooting 💫
    * 					eg: ? (help), > (next arrow), < (back arrow), need help ?
    */
 
-  if (!str.length) {
+  if (
+    isEmpty(str) ||
+    isNonDigitCharacter(str) ||
+    isSymbolicText(str) ||
+    isUnicodeOrPunctuation(str)
+  ) {
     return 0;
   }
 
-  // Step 1
-  const alphaNumericIconMap = [
-    'x', // close
-    'i' // info
-  ];
-  // Step 1a
-  if (alphaNumericIconMap.includes(str)) {
-    return 0;
-  }
+  return 1;
+}
+
+function isEmpty(str) {
+  return sanitize(str).length === 0;
+}
 
-  // Step 2
+function isNonDigitCharacter(str) {
+  return str.length === 1 && !str.match(/\d/i);
```

Comment:
the `i` isn't needed since digits can't be capitalized. Could also use `str.match(/\D/)` instead of negating it, but that's up to you (both are fine).

```suggestion
  return str.length === 1 && !str.match(/\d/);
```

Reply:
Addressed in c4aa0620